src/datasource/zabbix/proxy/cachingProxy.ts

```diff
@@ -100,9 +100,25 @@ function cacheRequest(func, funcName, funcScope, self) {
   };
 }
 
+//TODO: add type for the args
 function getRequestHash(args) {
-  const argsJson = JSON.stringify(args);
-  return getHash(argsJson);
+  try {
+    if (typeof args === 'object') {
+      args = Array.from(args || {}).map((arg: any) => {
+        if (arg?.scopedVars?.__sceneObject) {
+          const { scopedVars, ...rest } = arg;
+          const { __sceneObject, ...scopedVarsWithoutScenes } = scopedVars;
+          return { ...rest, scopedVars: scopedVarsWithoutScenes };
+        }
+        return arg;
+      });
+    }
+    const argsJson = JSON.stringify(args);
+    return getHash(argsJson);
+  } catch (ex) {
+    console.error(`failing to get the request hash for caching. type of args ${typeof args}`);
+    return 0;
```

Comment:
i wonder if it would be safer to return `Math.random()` or something similar here, because `0` will cause every such request to share the cache-key...  WDYT?